css/index.css

```diff
@@ -1,139 +1,287 @@
 }
 
 h2 {
   color: #652613;
-  border-top: 10px solid rgb(91, 131, 118);
-  padding-top: 50px;
+  border-top: 3px solid rgb(91, 131, 118);
+  padding-top: 1em;
   text-align: center;
-  font-size: 40px;
-  font-family: Baskerville bold;
+  font-size: 2rem;
 }
 
-p {
-  margin: 0px 300px 50px;
-  padding: 10px;
-  font-family: Baskerville;
-  text-indent: 3em;
-  border-radius: 4px;
-  font-size: 20px;
+.container {
+  width: 100%;
+  margin: 0 auto;
+  text-indent: 2em;
   text-align: justify;
 }
 
-img.image1 {
-  width: 30%;
-  height: 300px;
+.container > p {
+  width: 60%;
+  margin: 1em auto;
 }
 
-div.scroll-container {
-  background-color: #333;
-  overflow: auto;
-  white-space: nowrap;
-  padding: 10px;
-}
-
-div.scroll-container img {
-  padding: 10px;
-}
+img {
+  max-width: 100%;
+  display: block;
+} 
 
-nav {
-    position: fixed;
-    top: 0;
-    right: 0;
-    left: 0;
-    z-index: 999;
+.image1 {
+  width: 100%;
+  height: auto;
 }
 
-.navbar {
-    background-color: #2a3d39;
-    padding: 20px;
-    display: flex;
-    flex-direction: column;
+.scroll-container {
+  background-color: #304540;
+  overflow: auto;
+  white-space: nowrap;
+  padding: 0.1em;
 }
 
-.navbar > a {
-    font-size: 20px;
-    color: #EDE0D4;
-}
+.scroll-container img {
+  padding: 5px;
+} 
 
 .experience {
-    display: flex;
-    flex-direction: column;
-    margin: 0px 100px 50px;
-    padding: 10px;
-    border-radius: 4px;
-    font-family: Arial, Helvetica, sans-serif;
-    font-size: 18px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 2em 1em;
+  padding: 0.2em;
 }
 
 .exp-one {
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
 }
 
 .pos-one {
-    font-weight: 600;
+  font-weight: 600;
 }
 
 .year-one {
-    margin-left: auto;
-    font-weight: 600;
+  margin-left: auto;
+  font-weight: 600;
 }
 
 .descr-one {
-    margin-right: auto;
-    padding: 10px 100px 10px 0px;
-    text-align: justify;
+  margin-right: auto;
+  padding: 0.1em 4em 0.1em 0;
+  text-align: justify;
+  text-indent: 2em;
 }
 
 .exp-two {
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    margin-top: 50px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 1em;
 }
 
 .pos-two {
-    font-weight: 600;
+  font-weight: 600;
 }
 
 .year-two {
-    margin-left: auto;
-    font-weight: 600;
+  margin-left: auto;
+  font-weight: 600;
 }
 
 .descr-two {
-    margin-right: auto;
-    padding: 10px 90px 10px 0px;
-    text-align: justify;
+  margin-right: auto;
+  padding: 0.1em 4em 0.1em 0;
+  text-align: justify;
+  text-indent: 2em;
 }
 
 .links {
-    background-color: #7F5539;
-    padding: 30px 200px;
-    display: flex;
+  background-color: #7f5539;
+  /* padding: 30px 100px; */
+  padding: 2em 3em;
+  display: flex;
 }
 
 .links > a {
-    color: #EDE0D4;
-    font-size: 18px;   
+  color: #ede0d4;
+  /* font-size: 18px; */
```

Comment:
In general, it's a good idea to remove any commented out code before submitting a PR, unless it's being left in for a specific reason. Not a big deal though.